data/data-pipeline/data_pipeline/score/score_m.py

```diff
@@ -182,7 +196,9 @@ def _climate_factor(self) -> bool:
             & self.df[field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES]
         )
 
-        self._increment_total_eligibility_exceeded(climate_eligibility_columns)
+        self._increment_total_eligibility_exceeded(
+            climate_eligibility_columns, skip_fips=("72")
```

Comment:
i'm sorry it's a string not a constant. all of them could in theory have different fips codes, etc. not sure where to put this.

Reply:
That's fine for now. Eventually it could be something like skip_fips=constants.PR_FIPS_CODE or something. Not today.